common/djangoapps/enrollment/views.py

```diff
@@ -311,48 +337,88 @@ class EnrollmentListView(APIView, ApiKeyPermissionMixIn):
                 * name: Name of the attribute
                 * value: Value of the attribute
 
+            * is_active: Optional. A Boolean value that indicates whether the
+              enrollment is active. Only server-to-server requests can
+              deactivate an enrollment.
+
+            * mode: Optional. The course mode for the enrollment. Individual
+              users cannot upgrade their enrollment mode from "honor". Only
+              server-to-server requests can enroll with other modes.
+            
+            * user: Optional. The user ID of the currently logged in user. You
+              cannot use the command to enroll a different user.
+
         **Response Values**
 
-            A collection of course enrollments for the user, or for the newly created enrollment.
-            Each course enrollment contains:
+            If honor mode is not supported for the course, an HTTP x "Message"
```

Comment:
@srpearce looking at the code, the lack of an honor mode isn't something we necessarily check for. Here are the error states:

*GET*
400: An error occurred while retrieving enrollments (not sure exactly what went wrong)
404: Invalid permission to view enrollment data for user

*POST*
400: No course_id specified, or course does not exist; or, invalid is_active status
403: Upgrade attempted by a non-admin user
406: User does not exist